doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -97,7 +97,71 @@ It covers the test cases against networking management operations.
 
 
 #### 4.3.4.6 Compute - Nova
-It covers the test cases against compute management operations.
+
+Nova API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`